packages/core/src/external/fhir/shared/namespaces.ts

```diff
@@ -0,0 +1 @@
+export const IETF_URL = "urn:ietf:rfc:3986";
```

Comment:
nit: would `IETF_URN` be better here?

when I see a URL const, I'm expecting to see an `http` type URL

Reply:
This is the namespace for the IETF's URI standard, so I think it should be named accordingly.

URI, though, not URL - I should update that.

Reply:
https://terminology.hl7.org/3.1.0/NamingSystem-uri.html